pkgs/applications/video/media-downloader/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, qt5
+, ffmpeg-full
+, aria2
+, yt-dlp
```

Comment:
You said in the description it is a frontend for youtube-dl.
And here you are using an specific fork.
Any reason for that?

Reply:
All description was copy from upstream, you can check the link see the detail.

Reply:
Understood.